mountpoint-s3/src/cli.rs

```diff
@@ -728,11 +769,11 @@ where
     if let Some(file_mode) = args.file_mode {
         filesystem_config.file_mode = file_mode;
     }
-    filesystem_config.storage_class = args.storage_class;
+    filesystem_config.storage_class = args.storage_class.clone();
```

Comment:
nit: revert? (probably the idea was to reject express cache when the mounted bucket is also on express?)

Reply:
no, the `.clone()` is to avoid invalidating `args`, which is required later on. 

But you are right we should also consider the storage class when we add validation for the shared cache!